### PR TITLE
[Docs-2.0] Update ELI5.md

### PR DESCRIPTION
**Pull request by BTCparadigm**
Update ELI5 to WW2

### CODE REVIEW

File: docs/using-wasabi/ELI5.md [hunk 1]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
 
 ## CoinJoin with Wasabi
 
-If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choise because it by default coinjoins your coins, while braking the link with it's transaction history.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:61]:
```suggestion
If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choice because, by default, it coinjoins your coins, thus breaking it's transaction history link.
```

File: docs/using-wasabi/ELI5.md [hunk 2]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 1M sat after few minutes have passed from the transaction being confirmed.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:63]:
```suggestion
By default, the wallet automatically starts coinjoining UTXO's larger than 0.01 BTC, after a few minutes have passed since the transaction was confirmed. 
```
might be better to use BTC instead of sat because _sat_ nomination is not yet implemented.

File: docs/using-wasabi/ELI5.md [hunk 3]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choise because it by default coinjoins your coins, while braking the link with it's transaction history.
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 1M sat after few minutes have passed from the transaction being confirmed. 
+At this point, Wasabi will deduct  0,3% of coordinator fees from that total UTXO balance and the necessary Bitcoin networks coinjoin transaction fees.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:64]:
```suggestion
At this point, Wasabi will deduct  0,3% from that total UTXO balance as coordinator fee and the necessary Bitcoin network transaction fee. 
```

File: docs/using-wasabi/ELI5.md [hunk 4]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choise because it by default coinjoins your coins, while braking the link with it's transaction history.
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 1M sat after few minutes have passed from the transaction being confirmed. 
+At this point, Wasabi will deduct  0,3% of coordinator fees from that total UTXO balance and the necessary Bitcoin networks coinjoin transaction fees. 
+Once all or some of these coins are mixed, they will show up in the software's main view as "private coins".
+The wallet will then automatically mix the remaining coins or the change outputs that the user receives back after creating transactions, but the coordinator fees won't be charged from the user again, althought basic Bitcoin networks coinjoin transaction fees do apply.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:66]:
```suggestion
The wallet will then automatically mix the remaining coins or the change outputs that the user receives back after creating transactions, but the coordinator fees will not be charged from the user again. Although, Bitcoin network transaction fees still apply.
```

File: docs/using-wasabi/ELI5.md [hunk 5]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
 
-Now have a little patience, within a short while you should have a sufficient anonymity set to protect your privacy.
+In case the user has only smaller than 1M sat UTXO's, the wallet will which to Plebs Don't Pay -mode, which means it wont automatically coinjoin users coins.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:68]:
```suggestion
When the user only has UTXO's smaller than 0.01 BTC, the wallet will switch to _Plebs Don't Pay -mode_, which means it won't automatically coinjoin the users coins.
```

File: docs/using-wasabi/ELI5.md [hunk 6]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+Users will still be able to manually participate in coinjoins if they wish and they won't need to pay any coordinator fees.
 
-See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of CoinJoin, details on the Wasabi implementation and best practices how to use it properly.
+See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of coinjoin, details on the Wasabi implementation and best practices how to use it properly.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:71]:
```suggestion
See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of coinjoin, details on the Wasabi implementation and best practices on how to use it properly.
```

File: docs/using-wasabi/ELI5.md [hunk 7]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
 
 ## Sending bitcoin
 
-In the Wasabi `Send` tab you can give your bitcoins to someone else, or send them to another wallet of yours.
-1. Select the coins which you want to spend.
+In Wasabi, you can use `Send` button to give your bitcoins to someone else, or send them to another wallet of yours.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:75]:
```suggestion
In Wasabi, you can use the `Send` button to give your bitcoin to someone else or send them to another wallet of yours.
```

File: docs/using-wasabi/ELI5.md [hunk 8]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+1. Click `Send` button from top right corner of the main view.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:76]:
```suggestion
1. Click the `Send` button (top right corner) at the main view.
```

File: docs/using-wasabi/ELI5.md [hunk 9]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-3. Label the observers who know of this transaction.
-4. Specify the amount the receiver should get.
-5. Set the fee to select how fast you want the transaction to confirm.
-6. Verify that everything is correct.
-7. Type in your password.
-8. Hit the `Send` button.
-
-See [this chapter](/using-wasabi/Send.md) for more details on sending bitcoins and the privacy nuances of coin control.
+3. Specify the amount the receiver should get and click `Continue`.
+4. From `Advanced options` you can set a custom transaction fee, otherwise the wallet automatically estimates it.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:79]:
```suggestion
4. At `Advanced options` you can set a custom transaction fee, otherwise the wallet automatically estimates it.
```

File: docs/using-wasabi/ELI5.md [hunk 10]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+6. Verify that everything is correct and click `Confirm` or adjust the transaction.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:80]:
```suggestion
6. Check if everything is filled in correctly and click `Confirm`.
```

File: docs/using-wasabi/ELI5.md [hunk 11]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+7. Optionally you can click the shield image from the top right corner of the `Transaction Preview` to see suggestions on how to avoid generating a change output in this payment.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:81]:
```suggestion
7. Optionally, you can click the shield image (top right corner) at `Preview Transaction` to see suggestions on how to avoid generating a change output in this payment.
```

File: docs/using-wasabi/ELI5.md [hunk 12]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+8. Enter the name of the person or company receiving the payment and click `Continue`.
+9. Type in your password and click `Continue`.
+10. You should see a pop-up window telling you that the transaction was broadcasted to the Bitcoin network.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:84]:
```suggestion
10. You will now see a pop-up window telling you that the transaction is broadcasted to the Bitcoin network.
```

File: docs/using-wasabi/ELI5.md [hunk 13]
```diff
@@ -50,35 +50,37 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+
+See [this chapter](/using-wasabi/Send.md) for more details on sending bitcoins and the privacy nuances of it.
```
Comment (MarnixCroes) [inline docs/using-wasabi/ELI5.md:86]:
```suggestion
See [this chapter](/using-wasabi/Send.md) for more details about sending bitcoin and the privacy nuances of it.
```

**Review by MarnixCroes — COMMENTED** [review]
Small improvements

File: docs/using-wasabi/ELI5.md [hunk 14]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-3. Label the observers who know of this transaction.
-4. Specify the amount the receiver should get.
-5. Set the fee to select how fast you want the transaction to confirm.
-6. Verify that everything is correct.
-7. Type in your password.
-8. Hit the `Send` button.
-
-See [this chapter](/using-wasabi/Send.md) for more details on sending bitcoins and the privacy nuances of coin control.
+3. Specify the amount the receiver should get and click `Continue`.
+4. At `Advanced options` you can set a custom transaction fee, otherwise the wallet automatically estimates it.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:79]:
```suggestion
4. At `Advanced options` you can set a custom transaction fee rate, otherwise the wallet automatically estimates it.
```

File: docs/using-wasabi/ELI5.md [hunk 15]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+6. Check if everything is filled in correctly and click `Confirm`.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:80]: We should mention at `Preview Transaction` here.

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/ELI5.md [hunk 16]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 0.01 BTC, after a few minutes have passed since the transaction was confirmed. 
+At this point, Wasabi will deduct  0,3% from that total UTXO balance as coordinator fee and the necessary Bitcoin network transaction fee. 
+Once all or some of these coins are mixed, they will show up in the software's main view as "private coins".
+The wallet will then automatically mix the remaining coins or the change outputs that the user receives back after creating transactions, but the coordinator fees will not be charged from the user again. Although, Bitcoin network transaction fees still apply.
 
-Now have a little patience, within a short while you should have a sufficient anonymity set to protect your privacy.
+When the user only has UTXO's smaller than 0.01 BTC, the wallet will switch to _Plebs Don't Pay -mode_, which means it won't automatically coinjoin the users coins.
```
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:68]: Coordinator doesnt know links between users inputs so UTXO is correct IMO

File: docs/using-wasabi/ELI5.md [hunk 17]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
 
 ## Receiving bitcoin
 
-In order to receive bitcoins, you should generate an address in the `Receive` tab.
-But first, you must label it with the names of any observers who know that this address is yours.
+In order to receive bitcoin, you should generate an address in the `Receive` tab.
+But first, you must label it with the names of any observers who know about this transaction.
```
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:53]: True. Didnt figure out how to explain it shortly. Will figure something out.

File: docs/using-wasabi/ELI5.md [hunk 18]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
 
-If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choice because, by default, it coinjoins your coins, thus breaking it's transaction history link.
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 0.01 BTC, after a few minutes have passed since the transaction was confirmed.
```
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:63]: I think it is correct because of the delays before auto mixing starts, which was just merged. 
I think we should explain shortly the Plebs dont pay and manual coinjoining  because there is 2 ways the wallet works by default.

File: docs/using-wasabi/ELI5.md [hunk 19]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choice because, by default, it coinjoins your coins, thus breaking it's transaction history link.
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 0.01 BTC, after a few minutes have passed since the transaction was confirmed. 
+At this point, Wasabi will deduct  0,3% from that total UTXO balance as coordinator fee and the necessary Bitcoin network transaction fee. 
+Once all or some of these coins are mixed, they will show up in the software's main view as "private coins".
```
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:65]: Arent they under the main balance?
At least for me they are shown as "Private coins"

File: docs/using-wasabi/ELI5.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:53]: Why change something that was already correct and clear?

File: docs/using-wasabi/ELI5.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:63]: > I think it is correct because of the delays before auto mixing starts, which was just merged.

It is after a few minutes that the wallet was loaded not "after a few minutes have passed since the transaction was confirmed."

> I think we should explain shortly the Plebs Dont Pay and "manual" coinjoining because there is 2 ways the wallet works by default.

This is just a brief explanation how the wallet works, there is no need for details.

File: docs/using-wasabi/ELI5.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:65]: Do you see the coins themselves? or do you just see the total value of those coins?

File: docs/using-wasabi/ELI5.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:53]: Hmm. Not sure what was I thinking. Will change it back.

File: docs/using-wasabi/ELI5.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:63]: I was thinking about this from the first-time-user perspective but you are right in the case where user already has confirmed coins in the wallet.

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:63]: What is the other default way? I am not sure I get what you mean.

We can simply say "By default, the wallet automatically coinjoins your bitcoin."

File: docs/using-wasabi/ELI5.md [hunk 20]
```diff
@@ -49,36 +49,38 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choice because, by default, it coinjoins your coins, thus breaking it's transaction history link.
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining UTXO's larger than 0.01 BTC, after a few minutes have passed since the transaction was confirmed. 
+At this point, Wasabi will deduct  0,3% from that total UTXO balance as coordinator fee and the necessary Bitcoin network transaction fee.
```
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:64]: There is no need for these details.

If you look at the text that you are trying to edit, it didn't mention any detail, it just explained how to use coinjoin.

File: docs/using-wasabi/ELI5.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:65]: Talking about value of a coin might confuse newbies more by making them think of USD price or something but I changed it a bit so its hopefully clear now.

File: docs/using-wasabi/ELI5.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md]: > What is the other default way? I am not sure I get what you mean.

I mean, it either does automatically coinjoin users bitcoin or it doesnt, depending on how big UTXO's do they have.
It would be very confusing for user with less than 0.01 BTC if we said the wallet automatically coinjoins their coins even if it actually does not.

> We can simply say "By default, the wallet automatically coinjoins your bitcoin."

I agree we might not need to specify that starting the coinjoin takes a moment but I do think we have explain the limit.

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md]: This is just an ELI5 document, we shouldn't complicate things.
We should have documents where everything is explained in details.

**Review by yahiheb — CHANGES_REQUESTED** [review]
@BTCparadigm This is just an ELI5 document, we should keep it as simple as possible, there is no need to mention all the use cases here.
At the end of each section there is a link for more details if the reader wants to learn more how sending or conjoining work.

**Comment by BTCparadigm** [comment]
@yahiheb You are right, I went in to details.
I simplified sending process explanation but IMHO coinjoin part is ok, because it's good to mention wallets default behaviour differences with more or less than 0.01 BTC UTXO.

**Comment by yahiheb** [comment]
> but IMHO coinjoin part is ok, because it's good to mention wallets default behaviour differences with more or less than 0.01 BTC UTXO.

I don't agree with that. One or two sentences saying that coinjoin happens automatically by default with WW2 is better, maybe we can mention it is possible to turn that off.

The previous coinjoin explanation didn't mention any min amount to coinjoin and that was good for this document.

File: docs/using-wasabi/ELI5.md [hunk 21]
```diff
@@ -49,36 +49,33 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-## CoinJoin with Wasabi
+## Sending bitcoin
 
-If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
-Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+In Wasabi, you can use the `Send` button to give your bitcoin to someone else or send them to another wallet of yours.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:61]: IMO we should just remove this sentence.

```suggestion
```

File: docs/using-wasabi/ELI5.md [hunk 22]
```diff
@@ -49,36 +49,33 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+1. Click the `Send` button (top right corner) at the main view.
+2. Insert the address of the receiver.
+3. Specify the amount the receiver should get and click `Continue`.
+4. Make sure everything is filled in correctly in `Preview Transaction` window and click `Confirm`.
+5. Enter the name of the person or company receiving the payment and click `Continue`.
+6. Type in your password and click `Continue`.
+7. You will now see a pop-up window telling you that the transaction is broadcasted to the Bitcoin network.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:68]:
```suggestion

You will now see a pop-up window telling you that the transaction is broadcasted to the Bitcoin network.
```

File: docs/using-wasabi/ELI5.md [hunk 23]
```diff
@@ -49,36 +49,33 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
 
-Now have a little patience, within a short while you should have a sufficient anonymity set to protect your privacy.
+See [this chapter](/using-wasabi/Send.md) for more details about sending bitcoin and the privacy nuances of it.
 
-See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of CoinJoin, details on the Wasabi implementation and best practices how to use it properly.
+## Coinjoin with Wasabi
 
-## Sending bitcoin
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, using Wasabi is the obvious choice.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:74]: Just remove this IMO.
```suggestion
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/ELI5.md (hunk 23 above)
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:74]: Lol. After +46 comments for changing ~25 lines I'm at a point that I could commit any suggestions just to avoid arguing but I do think explanation for 5y old should include these basic reasonings, even if the kid doesnt know UTXO model. 
That sentence makes one learn.

File: docs/using-wasabi/ELI5.md [hunk 24]
```diff
@@ -49,36 +49,33 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining received UTXO's few minutes after the transaction is confirmed.
+Once all or some of these funds are properly coinjoined, the corresponding amount will show up in the software's main view as "private coins".
+The wallet will then automatically coinjoin the remaining coins or the change outputs that the user receives back after creating transactions, but the coordinator fees will not be charged from the user again.
```
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:78]: I think this suggestion is confusing. 
Nothing indicates that all coins might not get mixed in one round, making the user wonder why the amount in `private coins` doesnt move much.
I tried avoid using the word "hop" and to explain what it means in a way that a 5y old might understand.
Do you have strong feelings about this one?

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:78]: > Nothing indicates that all coins might not get mixed in one round, making the user wonder why the amount in `private coins` doesnt move much.

Nothing in your suggestion indicate that either.

> I tried avoid using the word "hop" and to explain what it means in a way that a 5y old might understand.

I don't think that a 5y old would understand what a UTXO or a confirmed transaction is.
Also the wallet does not automatically start coinjoining received UTXO's few minutes after the transaction is confirmed.

File: docs/using-wasabi/ELI5.md (hunk 23 above)
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:74]: We could avoid the back and forth by keeping things very simple and precise and changing only what really needs to be changed.

So saying `using Wasabi is the obvious choice` can be misleading, because if you use Wasabi without coinjoining then that will not help you hide what you do with your money from your employer.

The previous sentence `then you should CoinJoin your bitcoin, as this breaks the transaction history` was more precise and clear.

File: docs/using-wasabi/ELI5.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:78]: > Nothing in your suggestion indicate that either.

This part is supposed to do that. `Once all or some of these funds are properly coinjoined...`

> wallet does not automatically start coinjoining received UTXO's few minutes after the transaction is confirmed

How come?

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:78]: > These were supposed to do that. `Once all or some of these funds are properly coinjoined...` and `wallet will then automatically coinjoin the remaining coins`.

Nothing in those mention anything about rounds.

`Once all or some of these funds are properly coinjoined` for me this sentence talks about the received UTXO's that the wallet automatically started coinjoinig in the precedent sentence, so it does not talk about all the coins.

> > wallet does not automatically start coinjoining received UTXO's few minutes after the transaction is confirmed
> 
> How come?

There are many details when and how the wallet automatically starts coinjoining, but there is nothing that makes confirmed UTXOs wait few minutes.

> IMHO, your comment suggestion gives an impression that everything should get mixed in 1 coinjoin round.

My comment is not to explain how coinjoin works in detail.

File: docs/using-wasabi/ELI5.md (hunk 23 above)
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md]: "breaks the transaction history" sounds wrong. Edited.

File: docs/using-wasabi/ELI5.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:78]: > To me your version gives an impression that everything happens in one coinjoin.

My opinion is just to say how coinjoin works in WW2 without any detail.

> Delay AutoCoinJoin [zkSNACKs/WalletWasabi#7122 (comment)](https://github.com/zkSNACKs/WalletWasabi/issues/7122#issue-1115037094)

`The idea is to wait a few minutes after the wallet loaded and then start mixing.`

This is delay is after you open the wallet, it has nothing to do with confirmed UTXOs.

[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:78]: > Wait a while before the wallet starts mixing automatically.

From this and from general concept for delays, I would assume the delay is also for received coins. 
The reasoning is the same than when delaying after opening the wallet or when in `send` tab - user might want to do something with these coins so we should wait a moment. 
Not sure if that's how it's implemented though.
Ping @molnard ?

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/ELI5.md:78]: Even if all what you are saying is correct we don't need to mention all of that, it is just details and nuance for this specific document.

[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md:78]: Lets leave this one unresolved for a moment
https://github.com/zkSNACKs/WalletWasabi/issues/7122#issuecomment-1033173396

[the remark this answers is not in the document]
Comment (BTCparadigm) [reply docs/using-wasabi/ELI5.md]: Meh, updated the section anyway. We can make an addition later about the delay if needed and if the delay concept is solid.

File: docs/using-wasabi/ELI5.md [hunk 25]
```diff
@@ -49,36 +49,32 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-1. Go to the `CoinJoin` tab.
-2. Select the coins you want to make private.
-3. Type in your password.
-4. Click enqueue.
+1. Click the `Send` button (top right corner) at the main view.
+2. Insert the address of the receiver.
+3. Specify the amount the receiver should get and click `Continue`.
+4. Make sure everything is filled in correctly in `Preview Transaction` window and click `Confirm`.
+5. Enter the name of the person or company receiving the payment and click `Continue`.
+6. Type in your password and click `Continue`.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:66]: This is not how send works now.

File: docs/using-wasabi/ELI5.md [hunk 26]
```diff
@@ -49,36 +49,32 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
-2. Insert the address of the receiver.
-3. Label the observers who know of this transaction.
-4. Specify the amount the receiver should get.
-5. Set the fee to select how fast you want the transaction to confirm.
-6. Verify that everything is correct.
-7. Type in your password.
-8. Hit the `Send` button.
-
-See [this chapter](/using-wasabi/Send.md) for more details on sending bitcoins and the privacy nuances of coin control.
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should coinjoin your bitcoin, as this breaks link to its transaction history.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:74]:
```suggestion
If you don't like that your employer knows where you spend your money, or that a merchant can find out how much money you have, then you should coinjoin your bitcoin, as this breaks link to its transaction history.
```

File: docs/using-wasabi/ELI5.md [hunk 27]
```diff
@@ -49,36 +49,32 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+Although the coinjoin protocol is a bit complex, it is very easy to use in Wasabi.
+By default, the wallet automatically starts coinjoining received UTXO's. 
+Once a coin achieves enough ambiguity, the corresponding amount will show up in the software's main view as "private coins".
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:77]:
```suggestion
Once a coin achieves enough anonymity, the corresponding amount will show up in the software's main view as "private coins".
```

File: docs/using-wasabi/ELI5.md [hunk 28]
```diff
@@ -49,36 +49,32 @@ See [this chapter](/using-wasabi/WalletGeneration.md) for more information on ho
...
+The wallet will also automatically coinjoin the change output that the user gets back after creating a transaction, but the coordinator fees will not be charged again.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:78]: This is confusing, it is like coordinator fees are free for any change transaction which is not the case.

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/ELI5.md [hunk 29]
```diff
@@ -53,16 +53,19 @@ In order to receive bitcoin, you should generate an address in the `Receive` tab
 But first, you must label it with the names of any observers who know that this address is yours.
 For example, if Alice pays you back for last nights pizza, then the label is `Alice`.
 This is an important feature for you and the wallet to know which coin is from whom, and it will help with your privacy in the future.
+Once this newly generated address receives some coins, Wasabi Wallet will automatically hide this used address, so you don't reuse it by accident.
+In case you want to coinjoin these coins, click Play in the main views musicbox and choose a fitting profile setting for your wallet.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:57]:
```suggestion
In case you want to coinjoin these coins, click Play in the main view's musicbox and choose a fitting profile setting for your wallet.
```

File: docs/using-wasabi/ELI5.md [hunk 30]
```diff
@@ -53,16 +53,19 @@ In order to receive bitcoin, you should generate an address in the `Receive` tab
...
+Sending process is different depending on the privacy of coins you have. Assuming you have enough private coins the sending process is the following:
+
 1. Click the `Send` button (top right corner) at the main view.
-2. Insert the address of the receiver.
-3. Specify the amount the receiver should get and click `Continue`.
+2. Insert the address of the receiver and type in the amount the receiver should get, then click `Continue`.
+3. Enter the name of the person or company receiving the payment and click `Continue`.
 4. Make sure everything is filled in correctly in `Preview Transaction` window and click `Confirm`.
-5. Enter the name of the person or company receiving the payment and click `Continue`.
 6. Type in your password and click `Continue`.
```
Comment (yahiheb) [inline docs/using-wasabi/ELI5.md:69]:
```suggestion
5. Type in your password and click `Continue`.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]
Good job @BTCparadigm

**Review by MarnixCroes — APPROVED** [review]
ACK
